builds/uninstalling-openshift-builds.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: ASSEMBLY
+[id="uninstalling-openshift-builds"]
+= Uninstalling {builds-shortname}
+include::_attributes/common-attributes.adoc[]
+:context: uninstalling-openshift-builds
+
+toc::[]
+
+As a cluster administrator, you can uninstall the {builds-title} Operator. Uninstalling the Operator does not automatically remove the custom resources (CRs) and {builds-shortname} components. 
+
+To uninstall, perform the following tasks:
+
+. Delete the custom resources (CRs) that were created by default when you installed the Operator.
++
+[CAUTION]
```

Comment:
Caution is not a supported admonition type. See the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions).